docs/source/tutorials/AfriSAR_AGB_Maps_1681.ipynb

```diff
@@ -0,0 +1,1381 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    URL = STAC_API_URL
</pre><p>I don't think we need to declare another variable here, we can just use STAC_API_URL in line 7:</p><p> </p><p>cat = Client.open(STAC_API_URL, headers=headers)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/140/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>